files/selinux_build_module_simple.sh

```diff
@@ -4,13 +4,13 @@ module_dir="$2"
 
 set -e
 
-cd $module_dir
-test -d tmp || mkdir tmp
+cd "$module_dir"
+mkdir -p tmp
 
-checkmodule -M -m -o tmp/${module_name}.mod ${module_name}.te
+checkmodule -M -m -o "tmp/${module_name}.mod" "${module_name}.te"
 package_args="-o ${module_name}.pp -m tmp/${module_name}.mod"
 if [ -s "${module_name}.fc" ]; then
     package_args="${package_args} --fc ${module_name}.fc"
 fi
 
-semodule_package ${package_args}
+semodule_package "${package_args}"
```

Comment:
This change behavior.  Assuming we do not enter the if branch:

#### Before

`semodule_package` got 4+ arguments:
1. `-o`
2. `${module_name}.pp`
3. `-m`
4. `tmp/${module_name}.mod`

With the issue that when `${module_name}` has spaces, the parameter 2 and 4 are split and this break.

#### Now

`semodule_package` get 1 arguments:
1. `-o ${module_name}.pp -m tmp/${module_name}.mod`

This does not feel right.

#### Proposal

Spiting each parameter in a variable might help:

```sh
# crappy name for the example
o="${module_name}.pp"
m="tmp/${module_name}.mod"

#[...]

if [ -s "${module_name}.fc" ]; then
  semodule_package -o "${o}" -m "${m}" --fc "${module_name}.fc"
else
  semodule_package -o "${o}" -m "${m}"
end
```

Reply:
Ah yeah I shouldn't have quoted that line. Only the `checkmodule` line needs quoting. Shell scripts considered harmful 😉

Reply:
Actually the `semodule_package` does need to be split also, let me do that.

Reply:
Fixed.